CMakeLists.txt

```
@@ -1539,8 +1539,10 @@ endif()
 # Installation preparation.
 #
 
+include(GNUInstallDirs)
+
 set(EVENT_INSTALL_CMAKE_DIR
-    "${CMAKE_INSTALL_PREFIX}/lib/cmake/libevent")
+  "${CMAKE_INSTALL_LIBDIR}/cmake/libevent")
```

```suggestion
    "${CMAKE_INSTALL_LIBDIR}/cmake/libevent")
```

Let's preserve the indent